internal/overlord/servstate/manager_test.go

```diff
@@ -1611,3 +1612,14 @@ func (s *S) TestStopRunningNoServices(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(taskSet, IsNil)
 }
+
+type fakeLogManager struct{}
+
+func (f fakeLogManager) PlanChanged(p *plan.Plan) {
+	logger.Noticef("fakeLogManager notified of PlanChanged")
+	logger.Debugf("new plan: %#v", p)
```

Comment:
If we're not testing logs (and we shouldn't be), should we just make these methods no-ops? Or does this help significantly in testing?